requirements/common.txt

```diff
@@ -181,3 +181,6 @@ pyldap==2.4.37
 
 # Install Python Social Auth
 social-auth-app-django==1.2.0
+
+# Needed for Two Factor authentication
+-r two-factor-auth.txt
```

Comment:
Should some of the requirements be auto-generated instead?

Reply:
You mean get the requirements from the setup.py of `two-factor-auth`?

Reply:
I meant, just put the ones that are directly imported. The rest is generated with `tools/update-locked-requirement`.

Reply:
Yeah, we should be able to make that change.  requirements/README.md and http://zulip.readthedocs.io/en/latest/dependencies.html explain how this works.

Reply:
Done.

Reply:
I checked that twilio can be indirectly added via the extra https://github.com/Bouke/django-two-factor-auth/blob/master/setup.py#L21-L25.

I can't find any use of `pysocks` in the code, nor in the requirements of `django-two-factor-auth`.

Reply:
Should it be `django-two-factor-auth[Call,SMS]==1.6.1`?

Reply:
I have tried that; it doesn't work. Updated the code.